.github/workflows/build.yml

```diff
@@ -15,4 +15,5 @@ jobs:
         node-version: 16
         registry-url: https://registry.npmjs.org/
     - run: npm ci
+    - run: npm run lint && npm run format
```

Comment:
I think this would be clearer if `format:check` was left as it is in `package.json`. `npm run format` sounds like it is going to format, not check format

Reply:
Sure lets do the same for the linting as well